src/styles/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  padding: 10px;
+
+  &__container {
+    display: flex;
+    gap: $gap;
+    flex-wrap: wrap;
+    list-style: none;
+    padding-left: 0;
+    width: (7 * $square-size) + 6 * $gap;
```

Comment:
What are 7 and 6? Try to avoid magic numbers

Reply:
fixed:)